FileGetContentsLoader.php

```diff
@@ -31,8 +31,12 @@ public function loadDocument($url)
 
             $streamContextOptions = array(
               'method'  => 'GET',
-              'header'  => "Accept: application/ld+json, application/json; q=0.9, */*; q=0.1\r\n",
-              'timeout' => Processor::REMOTE_TIMEOUT
+              'header'  => "Accept: application/ld+json, application/json; q=0.9, */*; q=0.1\r\n".
+                           // Fix F. Michel 2017-12-06 - Some Web API require a User-Agent.
+                           // E.g. MusicBrainz returns error 403 if there is none.
+                           "User-Agent: JsonLD.php\r\n",
+              'timeout' => Processor::REMOTE_TIMEOUT,
+              'ssl' => [ 'verify_peer' => true, 'verify_peer_name' => true, 'allow_self_signed'=> false ]
```

Comment:
If you want the SSL change to be included, please file a separate PR. I think it needs more discussion

Reply:
Ok. Basically this should not change anything to the default behaviour. But useful to have the line in particular if your want to allow self-signed certificates. 
Anyway I agree this should be in a separate PR.